included_software.md

```diff
@@ -20,8 +21,7 @@ The specific patch versions included will depend on when the image was last buil
   * 3.7
 * PHP - `PHP_VERSION`
   * 5.6 (default)
-  * 7.2
-  * 7.4
+  * 7.0
```

Comment:
what is this change about? old leftover?

Reply:
yup, from when we were considering deprecating.